templates/repository/conf/axis2/axis2.xml.erb

```diff
@@ -1,732 +1,734 @@
-            is the address to which this member is bound to.
-        -->
-        <!--
-        <parameter name="localMemberBindAddress">127.0.0.1</parameter>
-        -->
-
-        <!--
-        The TCP port used by this member. This is the port through which other nodes will
-        contact this member
-         -->
-            <parameter name="localMemberPort"><%= @clustering['local_member_port'] %></parameter>
-        <!--
-            The bind port of this member. The difference between localMemberPort & localMemberBindPort
-            is that localMemberPort is the one that is advertised by this member, while localMemberBindPort
-            is the port to which this member is bound to.
-        -->
-        <!--
-        <parameter name="localMemberBindPort">4001</parameter>
-        -->
-
-        <!--
-        Properties specific to this member
-        -->
-        <parameter name="properties">
-            <property name="backendServerURL" value="https://${hostName}:${httpsPort}/services/"/>
-            <property name="mgtConsoleURL" value="https://${hostName}:${httpsPort}/"/>
-            <property name="subDomain" value="<%= @clustering['sub_domain'] %>"/>
-        </parameter>
-
-        <!--
-        Uncomment the following section to load custom Hazelcast data serializers.
-        -->
-        <!--
-        <parameter name="hazelcastSerializers">
-            <serializer typeClass="java.util.TreeSet">org.wso2.carbon.hazelcast.serializer.TreeSetSerializer
-            </serializer>
-            <serializer typeClass="java.util.Map">org.wso2.carbon.hazelcast.serializer.MapSerializer</serializer>
-        </parameter>
-        -->
-
-        <!--
-           The list of static or well-known members. These entries will only be valid if the
-           "membershipScheme" above is set to "wka"
-        -->
-<%- if @clustering['membership_scheme'] == 'wka' -%>
-        <members>
-   <%- @clustering['wka']['members'].each do |member| -%>
-            <member>
-                <hostName><%= member['hostname'] %></hostName>
-                <port><%= member['port'] %></port>
-            </member>
-   <%- end -%>
-        </members>
-<%- end -%>
-
-        @value
-
-        <!--
-        Enable the groupManagement entry if you need to run this node as a cluster manager.
-        Multiple application domains with different GroupManagementAgent implementations
-        can be defined in this section.
-        -->
-        <groupManagement enable="false">
-            <applicationDomain name="wso2.as.domain"
-                               description="AS group"
-                               agent="org.wso2.carbon.core.clustering.hazelcast.HazelcastGroupManagementAgent"
-                               subDomain="worker"
-                               port="2222"/>
-        </groupManagement>
-    </clustering>
-</axisconfig>
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)

```

Comment:
update license.

Reply:
Not needed since only some of the configs where updated.